publish_reports/o.py

```
@@ -0,0 +1,86 @@
+
+            os.rename(file_path, new_path)
+
+            print(f"Renamed {file_path2} to {file_path3}")
+
+
+def mv_wd_errors():
+    reports_dir = os.path.join(os.path.dirname(__file__), 'reports')
+
+    errors_files = []
+
+    for root, dirs, files in os.walk(reports_dir):
```

Ruff (0.8.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)